sql-plugin/src/main/scala/org/apache/spark/sql/rapids/collectionOperations.scala

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
This file is moved from rapids to sql, because:
1) to match the path for this file in Spark
2) to be able to call private [sql] class like `AbstractDataType`, `ArrayType.simpleString`